### ISSUE TITLE
Is this still being maintained?

### ISSUE THREAD

**Issue by github-userx** [totgezockt/askfmProfileGrabber#2]
I was also wondering why You chose Java for this app and not python? Apologies for my noob questions.

**Comment by totgezockt** [comment]
Well there were no real reasons for it. I just learned Java in school and I needed it quick, so I started working on it. Also python is kind of a hassle on Windows...

**Comment by github-userx** [comment]
Interesting! What courses or studies did you learned Java for? University or high school?
Do you think Java is easier to learn than python?
How did you come up with the idea for a askFM Archiver? Did you use askFM yourself regularly?
i hope I don’t annoy you with my questions!
kind Regards!


28. Mai 2019, 08:03 von notifications@github.com <mailto:notifications@github.com>:

**Comment by totgezockt** [comment]
@github-userx I might have phrased this wrong; I didn't exactly study Java, but we worked with it in our school class (highschool mind you). We mostly created very simple applications. All the other knowledge came by myself due to the fascination with programming.
I wouldn't exactly say that Java is easier than Python, it was just the first thing I learned.
I needed this application for a friend of mine who wanted all of his/her photos from his/her account. And because there were so many posts over several years I tried this. Also I had not found any application like this online so I created it on my own, which would also verify that nothing else is done with the collected data.
